src/devices/Ws2812B/BitmapImageNeo3.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Bindings.Utils;
+
+namespace Iot.Device.Bindings.WS2812B
+{
+    /// <summary>
+    /// Special 24bit RGB format for Neo pixel LEDs where each bit is converted to 3 bits.
+    /// A one is converted to 110, a zero is converted to 100.
+    /// </summary>
+    public class BitmapImageNeo3 : BitmapImage
+    {
+        const int BytesPerComponent = 3;
+        const int BytesPerPixel = BytesPerComponent * 3;
+        const int ResetDelayInBytes = 30; // 100us @ 2.4Mbps
+
+        public BitmapImageNeo3(int width, int height)
+            : base(new byte[width * height * BytesPerPixel + ResetDelayInBytes], width, height, width * BytesPerPixel)
+        {
+        }
+
+        public void SetPixel(int x, int y, uint color) => SetPixel(x, y, new Color(color));
+        public void SetPixel(int x, int y, byte level) => SetPixel(x, y, new Color(level));
+
+        public override void SetPixel(int x, int y, Color c)
+        {
+            var offset = y * Stride + x * BytesPerPixel;
+            Data[offset++] = _lookup[c.G * BytesPerComponent + 0];
+            Data[offset++] = _lookup[c.G * BytesPerComponent + 1];
+            Data[offset++] = _lookup[c.G * BytesPerComponent + 2];
+            Data[offset++] = _lookup[c.R * BytesPerComponent + 0];
+            Data[offset++] = _lookup[c.R * BytesPerComponent + 1];
+            Data[offset++] = _lookup[c.R * BytesPerComponent + 2];
+            Data[offset++] = _lookup[c.B * BytesPerComponent + 0];
+            Data[offset++] = _lookup[c.B * BytesPerComponent + 1];
+            Data[offset++] = _lookup[c.B * BytesPerComponent + 2];
+        }
+
+        private static readonly byte[] _lookup = new byte[256 * BytesPerComponent];
+        static BitmapImageNeo3()
```

Comment:
nit: our coding guidelines require explicit `private`

Reply:
I don't think that is allowed in C#

Reply:
@maloo I meant this: `private static BitmapImageNeo3()` - that is allowed

Reply:
Tell that to the compiler ;)
[The almighty](https://stackoverflow.com/questions/4119365/static-constructor-in-c-sharp)